gui/src/options.cpp

```diff
@@ -3924,6 +3925,52 @@ void ChartGroupsUI::CompletePanel(void) {
   m_UIcomplete = TRUE;
 }
 
+// Return an array of IANA time zones configured in the operating system.
+wxArrayString getTimeZones() {
+    std::vector<std::string> tzs = {
+        "America/New_York",
+        "America/Chicago",
+        "America/Denver",
+        "America/Los_Angeles",
+        "Europe/London",
+        "Europe/Berlin",
+        "Europe/Moscow",
+        "Asia/Tokyo",
+        "Asia/Shanghai",
+        "Australia/Sydney",
+        "Pacific/Auckland"
+        // Add more as needed
+    };
+    // TODO: if C++ 20 is enabled, we can use std::chrono to get a list of timezones
+    // in a platform-independent way.
+    // Get the time zone database.
+    /*
+    const auto& tzdb = std::chrono::get_tzdb();
```

Comment:
@rgleason , if OpenCPN is compiled with c++ 20, then it will be straightforward to get a list of timezones in a platform independent way. But right now OpenCPN is compiled with c++ 17.
One option is to only support "Local Time" and "UTC" until OpenCPN is upgraded to use c++ 20.